tests/fast-integration/monitoring/client.js

```diff
@@ -14,41 +14,34 @@ function prometheusPortForward() {
 }
 
 async function getPrometheusActiveTargets() {
-    let responseBody = await get("/api/v1/targets?state=active");
-    return responseBody.data.activeTargets;
+    let path = "/api/v1/targets?state=active"
```

Comment:
There are missing semi-colons for the decalations of `path` and `url`. Same applies in the other functions.